README.md

```diff
@@ -12,14 +12,53 @@ Here you'll find starter code and more advanced example use cases.
 </div>
 
 
```

Comment:
Can we have a table of contents similar to https://github.com/vinta/awesome-python/edit/master/README.md? I imagine with increasing number of links, it'll get hard to scroll through the entire README https://stackoverflow.com/questions/11948245/markdown-to-create-pages-and-table-of-contents

Reply:
Oh yes, thanks for the reminder.. this was something I meant to add when we had more examples. The day has come!

Reply:
Added